src/SFA.DAS.EmployerCommitmentsV2.Web/Startup/AspNetStartup.cs

```diff
@@ -28,7 +31,8 @@ public void ConfigureServices(IServiceCollection services)
                 .AddDasMvc()
                 .AddUrlHelper()
                 .AddEmployerUrlHelper()
-                .AddMemoryCache();
+                .AddMemoryCache()
+                .SaveKeysToStackExchangeRedis(_configuration, Environment.IsDevelopment());
```

Comment:
I would conditionally call this extension method, because the second parameter is basically a way of telling it not to do anything, which is not very readable, even if it adds a couple of lines here.